tests/test_client.py

```diff
@@ -1056,6 +1056,25 @@ def test_chained_exceptions_with_explicit_cause(self):
             }
         ]
 
+    @pytest.mark.skipif(
+         sys.version_info < (3, 11),
+         reason="requires BaseException.add_note (Python 3.11 or higher)"
+    )
+    def test_notes(self):
+        e = Exception("exception")
+        e.add_note("exception note 1")
+        e.add_note("exception note 2")
+        self.client.notify(e)
+        assert self.sent_report_count == 1
+
+        payload = self.server.received[0]['json_body']
+        metadata = payload['events'][0]['metaData']
+        notes = metadata['notes']
```

Comment:
This is now in `exception notes`

```suggestion
        notes = metadata['exception notes']
```